aws/table_aws_cloudtrail_lake_query.go

```diff
@@ -0,0 +1,199 @@
+package aws
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudtrailLakeQuery(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudtrail_lake_query",
+		Description: "AWS CloudTrail Lake Query",
+		List: &plugin.ListConfig{
+			ParentHydrate: listCloudTrailEventDataStores,
```

Comment:
Do we want a parent hydrate here, or should a user need to enter an event data store ARN? How many event data stores can be in a region and how many queries can an event data store have? What's the API performance like if there are a large number of both?

Reply:
We can create five event source per region https://docs.aws.amazon.com/awscloudtrail/latest/userguide/WhatIsCloudTrail-Limits.html

Reply:
Where did you find that figure? On that page, I can see:
```
Resource | Default Limit | Comments
Trails per region | 5 | This limit cannot be increased.
Event data stores in CloudTrail Lake | 10 | This limit cannot be increased.
```

So does that mean 10 per trail, region, or account? Or is it 5 instead of 10?

Reply:
Sorry it is ten

Reply:
Sometimes they set limits high, even if the number of resources allowed per region or account is very small.

I'm still unclear on how many event data stores are allowed in CloudTrail overall, as per my previous question. Can you please find out if it's 10 per trail, region, or account?

Reply:
@cbruno10 We can create upto 10 Event Data Store per trail, I tried to create more than 10 Event Data Store but got the limit exceeded error as below.

```
EventDataStoreMaxLimitExceededException
The maximum number of event data stores, 10, has already been created. This limit applies to the total number of event data stores that you can create in all regions.
```